sync-todo/v2/client/flutter/pubspec.yaml

```diff
@@ -6,15 +6,15 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.17.0"
+  sdk: ">=2.17.5 <4.0.0"
```

Comment:
Just a Q to clarify for me - I'm not familiar with this syntax but it looks like we're limiting the SDK here to versions less than 4.0.0, but we haven't done that previously. Just wanted to make sure this is intended. (Same with dart in the pubspec.lock)

Reply:
Yep! Probably unnecessary, but it's what the Dart SDK team uses and suggested.